src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -113,48 +110,139 @@ To chat with New Relic AI about log errors:
-    alt="A screenshot that shows how to chat with AI in errors inbox"
-    src={aiAskAiinErrorsInbox}
-/>
-
-<figcaption>
-    To chat with AI in errors inbox, go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities)**</DoNotTranslate> > <DoNotTranslate>**Errors inbox**</DoNotTranslate> > and select a workload. 
-</figcaption>
-
 To investigate stack traces in errors inbox with New Relic AI:
 
 1. Go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities)**</DoNotTranslate> > <DoNotTranslate>**Errors inbox**</DoNotTranslate>.
 2. Select a workload.
-2. Look for the <DoNotTranslate>**Ask AI**</DoNotTranslate> button at the top of the log details panel. Our AI will provide a summary of the error. 
+3. Look for the <DoNotTranslate>**Explain this error**</DoNotTranslate> button embedded in the Stack trace  panel. Our AI will provide a summary of the error. 
+
+You can dig deeper by asking follow-up questions. For example, you can say: 
+
+```
+Hi, I have an Azure environment and am having issues with Redis being periodically unavailable. Can you tell me more about this error? How can I diagnose this issue? 
+
+```
+### Fill alerting gaps [#coverage-gaps]
+
+As your team and systems evolve, staying ahead of potential issues is crucial. Alerts are your early warning system, but keeping them up-to-date can be challenging. NRAI can help identify areas where you might be missing critical notifications.
+
+Chat with AI and ask:
+
+```
+Show me all of my entities that are not covered by an alert. 
+```
+
+```
+Show me my OpenTelemetry services that are not covered by an alert.
+```
+
+### Recommend alerts [#recommend-alerts]
+
+If you have APM, browser, or mobile applications that you want to monitor but don't know where to start, you can use NRAI. 
+
+To get recommended alert conditions:
 
-You can take your investigation further by telling our AI about your stack and asking follow up questions. For example, you can say: 
+ 1. Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities)** >**Alerts & AI**>**Alert Coverage Gaps**.
+ 2. Click **Add alerts** in the top right-hand corner.
+ 3. Select one or more recommended alerts, then click **Next**.
+ 4. Select an existing policy for your alerts, or create a new one and click **Activate conditions**.
+
+You can further investigate this by chatting with our AI. You can ask:
+
+```
+Recommend alerts for this application.
+```
+
+Or, let's say you're a product manager and you're not interested in data from staging environments; you only want to be alerted on applications in production, you can ask: 
+
+```
+Can you recommend alerts for all my APM applications with `production` in their name?
+```
+
+Or, if you run an ecommerce site and you know that you have a lot of customers in a particular region and want to make sure your hosts don't get overloaded, you can ask NRAI:
```

Comment:
I think it's e-commerce to maintain consistency with the use of it previously in the doc